news/2021-05-07-monthly-beatmapping-contest-may-2021.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Monthly Beatmapping Contests: May 2021"
+date: 2021-05-07 08:00:00 +0000
+---
+
+Try your hand at this month's slow-slider-themed contest with a chance to become osu!'s next *Elite Mapper*!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+May's [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) includes opportunities for [**osu!**](LINK), [**osu!taiko**](LINK), and [**osu!mania**](LINK) mappers. Maybe next month we'll cover every mode at the same time, who knows?
+
+## Schedule
+
+- **Beatmapping (2.5 weeks):** Contests are announced at the start of the month and beatmappers create their entries.
+- [**Screening (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#screening) Competitive entries are selected by experienced beatmappers for each mode.
+- [**Judging (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#judging) Competitive entries are judged by different experienced beatmappers for each mode.
+- [**Rewards:**](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) Winners are announced and prizes are redeemed, including fancy profile badges and the legendary *Elite Mapper* title!
+
+Expert mathematicians might notice that this whole schedule doesn't fit within a month. We announce results for each contest halfway through the following month, meaning May's contest results will be published mid-June.
+
+## Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+We have two goals here at MBC headquarters: to create mapping contests every month and to ensure those contests aren't repetitive. It's a tough act to balance, which is why we have **limitations** — mapping restrictions that submissions need to follow, like "straight sliders only" or "circles cannot be used" or "you must map with your eyes closed". We unfortunately haven't tried that last one yet.
```

Comment:
```suggestion
We have two goals here at MBC headquarters: to create mapping contests every month, and to ensure those contests aren't repetitive. It's a tough act to balance, which is why we have **limitations**—mapping restrictions that submissions need to follow—like "straight sliders only" or "circles cannot be used" or "you must map with your eyes closed". (We unfortunately haven't tried that last one yet.)
```

Reply:
Dislike this change.

Reply:
I don't think your change makes it read better - two em-dashes introduce one pause two many or instead delineate the mapping restrictions block which is not what happens when you actually read the paragraph out loud.

You can ask twenty different writers about pause commas and get sixty different responses. It probably isn't worth fretting too much over.

Reply:
Apologies if I in any way sounded rude in my previous comment, as that was not my intention. I was already irritated from other matters when responding I was responding to it, and I ended up taking the comment personally, which is obviously something I should not have done. I realise now that my was suggestion was unneccessary, and that I should have made my thought process more clear beforehand. That's on me. I'll try to be more clear and think a little more next time.

---

> I don't think your change makes it read better - two em-dashes introduce one pause too many or instead delineates the mapping restrictions block which is not what happens when you actually read the paragraph out loud.

When I was reading this sentence, I had interpreted the immediate bit after the em dash as intending to delineate on what a "limitation" is in other words. (I.e., I thought the em dash was intending to replace a comma and not a colon.) That's why I had suggested the change, thinking it was just a typo, which I realize now it's not.

> You can ask twenty different writers about pause commas and get sixty different responses. It probably isn't worth fretting too much over.

This, along with adding the parantheses, was suggested because I intended to make it "more readable" (at least in my head). It was a reccomendation that I should have made more clear was completely up to preference.

Reply:
gonna leave this one as is, i think it reads better :(